### PR TITLE
feat: add logic to show cross chain tx link in activity and success toast

### PR DESCRIPTION
**Pull request by juanmahidalgo**
image

image

image

### CODE REVIEW

File: webapp/src/modules/item/sagas.ts [hunk 1]
```diff
@@ -27,7 +27,12 @@ import {
 import { isNFTPurchase } from 'decentraland-dapps/dist/modules/gateway/utils'
 import { PurchaseStatus } from 'decentraland-dapps/dist/modules/gateway/types'
 import { isErrorWithMessage } from '../../lib/error'
-import type { StatusResponse } from 'decentraland-transactions/crossChain'
+import {
```
Comment (LautaroPetaccio) [inline webapp/src/modules/item/sagas.ts:30]: Would you mind instantiating the `AxelarProvider` using the async procedure?

File: webapp/src/components/ActivityPage/Transaction/TransactionDetail/TransactionDetail.tsx [hunk 2]
```diff
@@ -29,7 +29,18 @@ const getHref = (tx: Transaction) => {
...
-  const { asset, text, tx } = props
+  const { asset, text, tx, isCrossChain = false } = props
+  const [txLink, setTxLink] = useState(getHref(tx))
+  useEffect(() => {
+    ;(async () => {
+      if (isCrossChain) {
+        const { AxelarProvider } = await import(
+          'decentraland-transactions/crossChain'
+        )
+        setTxLink(await AxelarProvider.getTxLink(tx.hash))
```
Comment (LautaroPetaccio) [inline webapp/src/components/ActivityPage/Transaction/TransactionDetail/TransactionDetail.tsx:40]: The await is not required, right?
```suggestion
        setTxLink(AxelarProvider.getTxLink(tx.hash))
```

File: webapp/src/modules/item/sagas.ts (hunk 1 above)
Comment (juanmahidalgo) [reply webapp/src/modules/item/sagas.ts]: sure! updated 👍

File: webapp/src/components/ActivityPage/Transaction/TransactionDetail/TransactionDetail.tsx (hunk 2 above)
Comment (juanmahidalgo) [reply webapp/src/components/ActivityPage/Transaction/TransactionDetail/TransactionDetail.tsx:40]: good call, not needed!

**Review by LautaroPetaccio — APPROVED** [review]

**Pull request merged**